test/crawler_web/controllers/api/v1/keyword_controller_test.exs

```diff
@@ -3,7 +3,101 @@ defmodule CrawlerWeb.Api.V1.KeywordControllerTest do
 
   alias Crawler.Keyword.Keywords
 
-  describe "POST /keyword/upload" do
+  describe "GET index/2" do
+    test "given a valid user, returns uploaded keywords", %{conn: conn} do
```

Comment:
should we test for `given an unauthenticated user`

Reply:
Added in [f8993ba](https://github.com/vnntsu/elixir-ic/pull/46/commits/f8993ba360e5baffcf585fe5ac362bcfbb3f53e6)